components/subventions-association-section/index.tsx

```diff
@@ -0,0 +1,79 @@
+'use client';
+
+import { Tag } from '#components-ui/tag';
+import { DataSubvention } from '#components/administrations';
+import { DataSectionClient } from '#components/section/data-section';
+import { FullTable } from '#components/table/full';
+import { EAdministration } from '#models/administrations/EAdministration';
+import { IAssociation } from '#models/core/types';
+import { ISession } from '#models/user/session';
+import { formatCurrency } from '#utils/helpers';
+import { useAPIRouteData } from 'hooks/fetch/use-API-route-data';
+
+export const SubventionsAssociationSection: React.FC<{
+  uniteLegale: IAssociation;
+  session: ISession | null;
+}> = ({ uniteLegale, session }) => {
+  const subventions = useAPIRouteData(
+    'subventions-association',
+    uniteLegale.siren,
+    session
+  );
+  if (!subventions) return null;
+
```

Comment:
Not sure about this. Either the user has the rights to see it and he/she prefer an explanation like : `Aucune demande de subvention n’a été trouvée pour cette association.`. Or h/she is not agent connected and we could display the `<AgentWall />` component